src/current/v22.2/backup.md

```diff
@@ -177,6 +177,34 @@ Improve the speed of backups to Azure Storage by increasing `cloudstorage.azure.
 
 **Default:** `1`
 
+#### Cloud storage cluster settings
+
+The following cluster settings limit the read and write rates to [cloud storage]({% link {{ page.version.version }}/use-cloud-storage.md %}). A user may choose to use these settings if their backups overwhelm the network. These settings limit throughput and as a result backups will take longer. The designated `<provider>`s include `s3`, `gs`, and `azure`.     
+
+#### `cloudstorage.<provider>.write.node_rate_limit`
+
+Limit the number of bytes per second per node across operations writing to the designated cloud storage provider if non-zero.
+
+**Default:** `0 B`
```

Comment:
Would it be less confusing to call the default unlimited/no-limit ?

Reply:
the data type is byte-size so I think it would make sense to include the `0 B`, but perhaps we can write "unlimited, `0 B`"